src/algorithm/evaluation.py

```diff
@@ -636,7 +652,15 @@ def prepare_explanation_table(self):
             table.add_column("Marking Scheme")
         self.explanation_table = table
 
-
+def get_evaluation_symbol(evaluation_meta,field_label,field_value):
+    if evaluation_meta["questions_meta"][field_label]["bonus_type"] == "BONUS_ON_ATTEMPT":
+        return "+"
+    if evaluation_meta["questions_meta"][field_label]["delta"]>0:
+        return "+"
+    if evaluation_meta["questions_meta"][field_label]["delta"]<0:
+        return "-"
+    else:
+        return "o"
```

Comment:
take question_meta as the argument -
evaluation_meta["questions_meta"][field_label] -> question_meta